.nvmrc

```diff
@@ -0,0 +1 @@
+14
```

Comment:
`.nvmrc` files are handy for developers who use [nvm](https://github.com/nvm-sh/nvm) to install multiple versions of node. You can just do `nvm use` before running any commands.